docs/reference_dataset/bosh-sample-hello-world.md

```diff
@@ -0,0 +1,194 @@
+```
+
+## The config files
+
+* [ci-deployment-overview.yml](/docs/reference_dataset/config_repository/hello-world/bosh-sample/ci-deployment-overview.yml)
+* [nginx](/docs/reference_dataset/config_repository/hello-world/bosh-sample/nginx)
+  * [enable-deployment.yml](/docs/reference_dataset/config_repository/hello-world/bosh-sample/nginx/enable-deployment.yml)
+  * [secrets](/docs/reference_dataset/config_repository/hello-world/bosh-sample/nginx/secrets)
+    * [meta.yml](/docs/reference_dataset/config_repository/hello-world/bosh-sample/nginx/secrets/meta.yml)
+    * [secrets.yml](/docs/reference_dataset/config_repository/hello-world/bosh-sample/nginx/secrets/secrets.yml)
+
+## The template files
+
+* [bosh-sample-versions.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/bosh-sample-versions.yml)
+* [nginx](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx)
+  * [deployment-dependencies.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/deployment-dependencies.yml)
+  * [template](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template)
+    * [adding-ntp-release-operators.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/adding-ntp-release-operators.yml)
+    * [nginx-tpl.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/nginx-tpl.yml)
+    * [ntp-release-vars.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/ntp-release-vars.yml)
+    * [openstack](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/openstack)
+      * [nginx-operators.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/openstack/nginx-operators.yml)
+    * [post-deploy.sh](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/post-deploy.sh)
+    * [pre-deploy.sh](/docs/reference_dataset/template_repository/hello-world/bosh-sample/nginx/template/pre-deploy.sh)
+* [template](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template)
+  * [cloud-config.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template/cloud-config.yml)
+  * [deploy.sh](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template/deploy.sh)
+  * [openstack](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template/openstack)
+    * [disk-types-cloud-operators.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template/openstack/disk-types-cloud-operators.yml)
+  * [runtime-config.yml](/docs/reference_dataset/template_repository/hello-world/bosh-sample/template/runtime-config.yml)
+
+## Required pipeline credentials for bosh-sample
+
+### bosh-sample-cf-apps-generated.yml
+
+No credentials required
+
+### bosh-sample-concourse-generated.yml
+
+No credentials required
+
+### bosh-sample-generated.yml
```

Comment:
is it possible, to also list pipelines using a property, to help identifying properties shared across pipelines